Uzira.rte/Actors/Brains/Uzira/Uzira.lua

```diff
@@ -0,0 +1,80 @@
+function Create(self)
+	self.minion = CreateAHuman("Skeleton", "Uzira.rte");
+	self.weapons = {"Uzira.rte/Blunderpop", "Uzira.rte/Blunderbuss", "Uzira.rte/Musket"};
+	self.spawnRadius = math.max(FrameMan.PlayerScreenWidth, FrameMan.PlayerScreenHeight) * 0.1;
+	self.boundsRadius = self.spawnRadius * 4 + self.minion.Radius;
+	self.spawnTerrainTolerance = 70;
+	self.spawnCost = self.minion:GetGoldValue(0, 1, 1);
+	
+	self.baseSpawnTime = 1000;
+	self.spawnTime = 1000;
+	self.spawnTimer = Timer();
+	self.minion.Team = self.Team;
+	self.minion.PinStrength = self.minion.Mass;
+	self.minion:AddScript("Uzira.rte/Actors/Shared/Undead.lua");
+	self.minions = {};
+end
+function Update(self)
+	if self.spawnTimer:IsPastSimMS(self.spawnTime) then
+		self.spawnTimer:Reset();
+		self.spawnTime = self.baseSpawnTime;
+		local funds = ActivityMan:GetActivity():GetTeamFunds(self.Team);
+		for i = 1, #self.minions do
+			local minion = self.minions[i];
+			if not MovableMan:IsActor(minion) then
+				table.remove(self.minions, i);
+			elseif minion.Health > 0 and SceneMan:ShortestDistance(self.Pos, minion.Pos, SceneMan.SceneWrapsX).Magnitude > self.boundsRadius then
+				minion.Health = 0;
+				for attachable in minion.Attachables do
+					if math.random() < 0.5 then
+						minion:RemoveAttachable(attachable, true, true);
+					end
+				end
+				--ActivityMan:GetActivity():ReportDeath(self.Team, -1);	--Disregard deaths of spawned minions?
```

Comment:
I think it's reasonable to disregard death of skeletons that died from being too far away, yeah.